util.go

```diff
@@ -18,3 +19,11 @@ func uuid() string {
 	uuid[6] = uuid[6]&^0xf0 | 0x40
 	return fmt.Sprintf("%x-%x-%x-%x-%x", uuid[0:4], uuid[4:6], uuid[6:8], uuid[8:10], uuid[10:])
 }
+
+func ReadClientVersion() string {
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		return "0.0.0"
+	}
+	return info.Main.Version
+}
```

Comment:
Any reason it's exported @christopheranderson?

Also, you're reading here the version of the "main" and not of "github.com/a8m/documentdb". I think this should be changed to something like:
```suggestion
func ReadClientVersion() string {
	info, ok := debug.ReadBuildInfo()
	if ok {
		for _, d := range info.Deps {
			if d.Path == "github.com/a8m/documentdb" {
				return d.Version
			}
		}
	}
	return "0.0.0"
}
```

Reply:
I could have probably just defined this in the one place it's used right now. In the other drivers, I've put it in utils since we also emit it in some places like error messages (though not uniformly, right now...)